helix-term/src/main.rs

```diff
@@ -73,6 +73,7 @@ FLAGS:
     -V, --version                  Prints version information
     --vsplit                       Splits all given files vertically into different windows
     --hsplit                       Splits all given files horizontally into different windows
+    +N                             Goto line number N, can be mentioned after each file
```

Comment:
Some programs give the `+N` before the filename like `less`'s `v` command. Kakoune only accepts one of these arguments and it only applies to the first file argument.

Reply:
Sure, we can do that, if you insist. But why? Given that we can give more flexible functionality here?

Reply:
It may simplify the code to only parse one `+` arg especially if it is allowed before the filename. This `+` syntax is only for the sake of compatibility with programs like `less` that use that syntax exclusively so there's no need to make it very flexible: you can already specify line and column with colons and numbers after the file names (for example `hx README.md:15:10`)

Reply:
Done. I looked at how NeoVim behaves and aligned to it.
It accepts as many +N arguments as you wish, but takes the latest one and applies to the first file in the list. Implementation is indeed a bit simpler and more straight forward that way.